MIGRATION-TO-SCK.md

```diff
@@ -0,0 +1,326 @@
+
+#### Pod Security Context
+If you are using the podSecurityContext option to set a pod security policy, you can use the podSecurityContext option in SCK-OTEL.
+
+#### Custom Annotations
+If you are using the customMetadataAnnotations option to set a custom annotation fields in your logs in SCK, you can use the "annotations" and "podAnnotations" option in SCK-OTEL.
+
+## Step 2: Delete SCK Deployment (Optional)
+If you want to delete your SCK deployment:
+
+* Get your current SCK deployment name by running this command:
+
+  * helm ls
+
+* **If you want to delete only logs and metrics**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete only logs**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete your entire SCK deployment (logs, objects and metrics), run the following command**
+
+  * helm delete local-k8s
+
+## Step 3: Installing SCK-OTEL
+
+* Ensure you have the necessary prerequisites for installing this application, as noted [here](https://github.com/splunk/sck-otel#prerequisites).
+* Set up Splunk to have a Splunk index, HEC and a HEC token ready to use, as noted [here](https://github.com/splunk/sck-otel#setup-splunk).
+* You can use SCK-OTEL as a non-root application on your kubernetes nodes - follow the steps [here](https://github.com/splunk/sck-otel#setup-for-non-root-user-group).
+* Install SCK-OTEL by using the .yaml file from Step 1 and follow the steps [here](https://github.com/splunk/sck-otel#deploy-with-helm-30).
+
+## Step 4: Check Data in Splunk
+* Check your logs index to see if you are receiving logs from your kubernetes cluster
+  * index="Your logs index"
+* Check your metrics index to see if you receiving metrics from your kubernetes cluster
+  * | mcatalog values(metric_name) WHERE index="Your metrics index"
+
+## Step 5: Delete SCK Deployment (Optional)
+### If you want to delete your SCK deployment
+* Get your current SCK deployment name by running this command:
+  * helm ls
+* **If you want to delete only logs and metrics**
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+    * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+* **If you want to delete only logs**
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+    * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+* **If you want to delete your entire SCK deployment (logs, objects and metrics), run the following command**
+  * helm delete local-k8s
```

Comment:
@chaitanyaphalak where does this content fit: https://github.com/splunk/sck-otel#search-for-splunk-connect-for-kubernetes-opentelemetry-metadata-in-splunk ?  I didn't notice it above.

Reply:
https://github.com/signalfx/splunk-otel-collector-chart/pull/277